NitroxPatcher/Patches/IngameMenu_OnSelect_Patch.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Reflection;
+using Harmony;
+using UWE;
+
+namespace NitroxPatcher.Patches
+{
+    public class IngameMenu_OnSelect_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(IngameMenu);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("OnSelect", BindingFlags.Public | BindingFlags.Instance);
+
+        public static void Postfix(IngameMenu __instance)
+        {
+            FreezeTime.End("IngameMenu");
```

Comment:
This change will also disable pausing in singleplayer. Can you add a check if multiplayer is running?

Reply:
This patch is not `Persistent`, meaning it'll be automatically injected when MP is started, and reverted when leaving a session.